internal/archiver/archiver.go

```diff
@@ -0,0 +1,67 @@
+package archiver
+
+import (
+	"os"
+	"strings"
+
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/fileutils"
+	"github.com/mholt/archiver/v3"
+)
+
+type FileMap struct {
+	Source string
+	Target string
+}
+
+func CreateTgz(filepath string, fileMaps []FileMap) error {
```

Comment:
Archiver supports creating archives, but its API is pointlessly convoluted. This package is just a wrapper that doesn't require us to write so much boilerplate just to create a simple archive.